pmlc/rt/jit_executable.cc

```diff
@@ -428,7 +458,8 @@ class JitExecutable final : public Executable {
     descriptors.clear();
     ptrs.clear();
 
-    std::copy(preParams.begin(), preParams.end(), std::back_inserter(ptrs));
+    network = abi.setupFunc(this->device.get());
```

Comment:
We need to pass all the user arguments to the setup function also.

Reply:
Actually, just the constant buffers need to be passed

Reply:
Definitely.  But I'd rather do that in a followup PR.

Reply:
I think we should call `setupFunc` in the constructor, not in `bindArguments`, but I suppose that can wait in a follow up.